docs/product-installation-guide/src/main/asciidoc/assembly_installing-on-eap.adoc

```diff
@@ -10,7 +10,7 @@ For information on how to install {EAP} 7.0.7 on your computer, see the https://
 include::eap-business-central-install-proc.adoc[leveloffset=+1]
 
 include::eap-execution-server-download-install-proc.adoc[leveloffset=+1]
-NOTE: This section describes configuring Business Central and the Execution Server in the same container. In production environments, they are usually run in different containers.
+NOTE: This section describes configuring Business Central and Execution Server in the same container. In production environments, they are usually run in different containers.
```

Comment:
This seems out of place. Supposed to be for eap-business-central-install-proc.adoc? Also, I'm not sure why this is in the assembly. Should be in the module, probably immediately after the intro sentence(s) and before the prereqs. From what I can tell.

Reply:
Sure @sterobin